public_html/wp-content/mu-plugins/blocks/source/blocks/organizers/content-select.js

```diff
@@ -12,7 +12,7 @@ import { __ } from '@wordpress/i18n';
 /**
  * Internal dependencies
  */
-import { buildOptions, ItemSelect, Option } from '../../components/item-select';
+import { ItemSelect, Option, buildOptions } from '../../components/item-select';
```

Comment:
`MAYBE` - It feels kind of odd to me that this is case-sensitive, but I don't feel too strongly.

Reply:
I went back & forth on that - we can ignore case if we want, but on the other hand if capitalization is an indicator of "type" (capital imports are components, while lowercase imports are helpers/settings), it does make sense to keep them separated somewhat.

Reply:
👍 

Keeping it case-sensitive is fine with me.